web-client/src/presenter/actions/addPenaltyInputAction.js

```diff
@@ -9,10 +9,10 @@ import { state } from 'cerebral';
  * @returns {void}
  */
 export const addPenaltyInputAction = ({ get, store }) => {
-  const { nameAcronym, penalties, subkey: penaltyType } = get(state.modal);
+  const { penalties, penaltyNameLabel, subkey: penaltyType } = get(state.modal);
```

Comment:
I thought we were changing this to penaltyTypeAcronym?

Reply:
Well, (Court) isn't an acronym anymore... idk, maybe penaltyNameSuffix?

Reply:
No last night while I was sleeping, It dawned on me why the variable name change again.  Silly me.